gym_solo/core/termination.py

```diff
@@ -40,8 +40,8 @@ def is_terminated(self) -> bool:
     termination conditions and value of _is_or attribute
     """
     
-    if not self._use_or:
-      raise ValueError('No termination condition other than OR is defined')
```

Comment:
This is something we really don't need. The user needs to define how the termination condition should be executed! That is with OR, AND, or something complex. We can add the functionality later if we want

Reply:
sgtm.